### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by shawwn**

### CODE REVIEW

**Comment by avaer** [comment]
Thanks, definitely needs a README.

However this project is a general browser -- it's not intended to have anything to do with servers, so the README probably shouldn't reference any specific server implementation.

**Comment by shawwn** [comment]
Makes sense. I trimmed the readme.

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (avaer) [inline yarn.lock]: Hm, would prefer to not commit `yarn.lock`. Might make merging repositories less clean.

**Comment by shawwn** [comment]
Whoops! Fixed.

**Comment by avaer** [comment]
Looks good ❤️.

Going forward I was thinking maybe all of these platform-specific repos should just be merged, benefitting from a common README? There are already 7 repos in the project:

https://github.com/modulesio/exokit // javascript/DOM parts
https://github.com/modulesio/exokit-bindings // native bindings
// platform-specific build stuff
https://github.com/modulesio/exokit-windows
https://github.com/modulesio/exokit-macos
https://github.com/modulesio/exokit-linux
https://github.com/modulesio/exokit-android
https://github.com/modulesio/exokit-ios

Would be nice to be able to `npm i exokit` on any platform and have it just work. And maybe have some way to not need to download deps like `glfw` in this case.

But those are separate issues, will open them. Thanks again!

**Pull request merged**